piff/des/decaminfo.py

```diff
@@ -144,12 +143,14 @@ def getPosition_chipnum(self, chipnums, ix, iy):
 
         :returns xPos, yPos:    Arrays of x and y coordinates in mm on the focal plane.
         """
-        # do getPosition but with chipnum instead
+        # do getPosition but with chipnum instead
+        if type(chipnums[0]) != int:
```

Comment:
If you just use `chipnums = np.array(chipnums, dtype=int)` once here at the top, can you eliminate the other type conversions below?

Reply:
Yeah, I think this is all simpler with just
```
chipnums = np.array(chipnums, dtype=int, copy=False)
```

Reply:
ok, and I changed the np.zeros(len(chipnums)) to np.zeros(chipnums.shape)